lib/src/test/java/org/team100/lib/targeting/InterceptTest.java

```diff
@@ -55,11 +56,11 @@ void testRobotMovingTargetStationary() {
                 new Translation2d(1, 1),
                 new GlobalVelocityR2(0, 0),
                 1);
-        assertTrue(azimuth.isPresent());
-        assertEquals(0, azimuth.get().getRadians(), DELTA);
+        assertTrue(azimuth.isEmpty());
```

Comment:
I think this is the same as the case above -- there's a solution at t=0.  Try a faster bullet, and/or allow solutions at t=0.